.github/workflows/config/spelling_allowlist.txt

```diff
@@ -214,12 +221,14 @@ prepend
 preprocessor
 programmatically
 pybind
+quakeDialects
```

Comment:
`Quake` is in the list use `Quake dialects` if required or escape with backticks.

Reply:
Fixed in [a3964e2](https://github.com/NVIDIA/cuda-quantum/pull/1763/commits/a3964e284910f99cbbddddfa7493747ab9735855).